scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -24,6 +26,8 @@ public static class PublishTargets
 
         private static string SharedFrameworkNugetVersion { get; set; }
 
+        private static List<string> _latestBlobs;
```

Comment:
Happy that we are moving away from `BuildTargetContext` 👍 It's Hard to keep track of variables as strings